.codespellrc

```diff
@@ -0,0 +1,3 @@
+[codespell]
+ignore-words-list = bloc,ser,dout,exten
```

Comment:
I think we should be able to drop this ignore list. I don't think it will be relevant in this repository.